packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -147,6 +147,7 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
                         onKeyDown={ (ev: React.KeyboardEvent<HTMLElement>) =>
                           this._navigateMonthEdge(ev, day.originalDate, weekIndex, dayIndex) }
                         aria-selected={ day.isSelected }
+                        aria-label={ day.originalDate.toLocaleDateString() }
```

Comment:
This seems IE11 specific; couldn't we just ask the user to pass this in? Or at least make sure that day.originalDate.toLocaleDateString exists before calling it?